### PR TITLE
[Xedra Evolved] Perpetual stew of bugfixes and QoL

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
Fixing bugs and adding random stuff that i found in my last playtest
#### Describe the solution
See the commits
#### Additional context
If you have some issues with the mod, its a good time to report them

### CODE REVIEW

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 1]
```diff
@@ -136,6 +136,7 @@
       "Your brain can't hold any more brilliance."
     ],
     "max_intensity": 3,
+    "apply_message": "Some great idea illuminated your mind, it's probably a good time to pick your laptop and do some calculations.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/effects/effects.json:139]:
```suggestion
    "apply_message": "A great idea has illuminated your mind, it's probably a good time to pick up your laptop and do some calculations.",
```

File: data/mods/Xedra_Evolved/items/inventor/gun.json [hunk 2]
```diff
@@ -84,13 +84,61 @@
       }
     ]
   },
+  {
+    "id": "ray_rifle",
+    "type": "GUN",
+    "reload_noise_volume": 4,
+    "name": { "str": "the hyperboloid" },
+    "description": "Pretty bulky, this laser rifle has double amount of lenses and emitters, which makes it really devastating after some modding.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/inventor/gun.json:92]:
```suggestion
    "description": "Pretty bulky, this laser rifle has double the amount of lenses and emitters, which makes it really devastating after some modding.",
```

File: data/mods/Xedra_Evolved/items/inventor/gun.json [hunk 3]
```diff
@@ -101,7 +149,7 @@
     "ammo": "aetheric_charge",
     "ammo_to_fire": 1,
     "range": 30,
-    "ranged_damage": { "damage_type": "heat", "amount": 35, "armor_penetration": 8 },
+    "ranged_damage": { "damage_type": "pure", "amount": 35 },
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/inventor/gun.json:152]: Legit choice :)

File: data/mods/Xedra_Evolved/items/inventor/gunmods.json [hunk 4]
```diff
@@ -53,6 +53,29 @@
     "range_modifier": -2,
     "min_skills": [ [ "weapon", 6 ], [ "deduction", 5 ] ]
   },
+  {
+    "id": "mod_inv_pistol_booster",
+    "type": "GUNMOD",
+    "name": { "str": "pistol's booster" },
+    "description": "A set of mechanisms, that alter the blocky pistol work enough to make it something in between of automatic pistol and a submachine gun.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/inventor/gunmods.json:60]:
```suggestion
    "description": "A set of mechanisms, that alter the action of the blocky pistol enough to make it something in between of automatic pistol and a submachine gun.",
```

File: data/mods/Xedra_Evolved/mutations/classes.json [hunk 5]
```diff
@@ -57,6 +57,6 @@
     "valid": false,
     "enchantments": [ "BONUS_INVENTOR" ],
     "cancels": [ "DREAMSMITH" ],
-    "vitamin_rates": [ [ "creative_spark", -604800 ] ]
+    "vitamin_rates": [ [ "creative_spark", -302400 ] ]
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/mutations/classes.json:60]: Is this 3 and a half days? Makes sense with the number of recipes.

File: data/mods/Xedra_Evolved/recipes/inventor/researches.json [hunk 6]
```diff
@@ -11,6 +11,7 @@
     "time": "2 h",
     "skill_used": "deduction",
     "difficulty": 2,
+    "flags": [ "BLIND_EASY" ],
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/recipes/inventor/researches.json:14]: If it requires the laptop how is it blind easy?  Or are you saying the laptop screen lights up enough to work from and it's just a weird side effect that blind easy means you can now use your laptop with a blindfold on?

**Review by Maleclypse — COMMENTED** [review]
Good changes! Thanks for all the work you do!

File: data/mods/Xedra_Evolved/recipes/inventor/researches.json (hunk 6 above)
Comment (GuardianDll) [reply data/mods/Xedra_Evolved/recipes/inventor/researches.json:14]: I thought blind easy refer only to the fact you can easily craft it at night, does it really allows you to craft while you blind?

**Comment by b3brodie** [comment]
Issue: The dreamforged armor recipes are all in the weapon crafting category instead of the armor one.  This is because they all copy from 'dreamforged_fullhelmet_ench' which has the 'CSC_XEDRA_WEAPONS' category when it should be 'CSC_XEDRA_ARMOR'

File: data/mods/Xedra_Evolved/recipes/inventor/researches.json (hunk 6 above)
Comment (Maleclypse) [reply data/mods/Xedra_Evolved/recipes/inventor/researches.json:14]: Went ahead and tested it, BLIND_EASY allows you to craft that recipe while wearing a blindfold.

Comment (GuardianDll) [reply data/mods/Xedra_Evolved/recipes/inventor/researches.json:14]: pretty dumb, but i don't think i can do much here - i just want to bring this aestetic of "researching till dawn"

**Comment by b3brodie** [comment]
Issue: From XEDRA Alchemy Perks: Potion of terrible strength has wrong spell cast id.  It has 'potion_strength' spell id, but this spell does not exist, throwing an error if its ever hovered over in the menu / used.  The actual spell id is 'terrible_strength'.

Issue: Same as potion of terrible strength, potion of wicked quick hands has the wrong spell id, 'potion_dex'.  It should be 'wicked_quick_hands'.

Related Issue: terrible_strength, wicked_quick_hands, and spell_hare_leap are spells that are cast via imbibing a potion, yet they have many definitions that don't seem to fit that.  For one, the strength and dexterity spells have a magus class which doesn't make sense.  Two, they have mana costs defined for them, even though they shouldn't cost anything to use other than the potion that the person consumes.  The strength and quick hands also are capable of being leveled up, which they should not be capable of having done.

**Comment by GuardianDll** [comment]
Now i am waiting two things: Male's decision about "should ending of the research remove the research recipe or not", and when Male send me spell_hyde_formula spell that they forgot to add into the mod

**Comment by b3brodie** [comment]
More alchemy bugs,  spiritshow_dust and charm_of_marzanna lack recipes.  This makes learning with the initiate alchemy perk to throw an error and potentially fail to give a recipe.

**Comment by Maleclypse** [comment]
> 

Those were already fixed in another PR I believe for the terrible strength and wicked quick

**Comment by GuardianDll** [comment]
ok, now the 65917 is merged, its okay to merge this one

**Pull request merged**